content/guides/tanzu-service-secret-sauce/index.md

```diff
@@ -0,0 +1,240 @@
+
+## Redis Deployment
+
+There are several options available to demonstrate the power of SecretTemplates; this guide will use [Redis](https://redis.io/) as its Guinea pig.  Redis is a good sample candidate as it is a very popular service that is used in many production applications.  It also has multiple distributions available for Kubernetes, but to the author’s knowledge, none support the Kubernetes service binding specification.  In this section, you will install a Redis Kubernetes operator, deploy an instance of a Redis cluster and database, and create a service binding compliant secret using a SecretTemplate.
+
+Installing the Redis operator is a relatively simple task as all the necessary resource definitions are contained in a single yaml file that is available at public URL.  You will use the [Redis Enterprise Cluster](https://docs.redis.com/latest/kubernetes/) operator in this guide.  To install the Redis operator, run the following commands:
+```sh
+kubectl create ns service-instances
+
+kubectl apply -f https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/v6.2.10-45/bundle.yaml -n service-instances
+```
+
+Validate the operator is installed with the following command:
+```sh
+kubectl get pods -l name=redis-enterprise-operator -n service-instances
+```
+
+You should see an output similar to the following:
+```text
+NAME                                         READY   STATUS    RESTARTS   AGE
+redis-enterprise-operator-74849d8c69-pnqvq   2/2     Running   0          39s
+```
+
+Now that the operator is deployed, you can create a Redis cluster and database.  To deploy the cluster and database, run the command below.  The configuration referenced in the following command also deploys a SecretTemplate resource to convert the subsequent Redis configuration into a service binding compliant secret:
+```sh
+ytt -f https://raw.githubusercontent.com/gm2552/redis-secret-template/main/templates/redisEnterpriseClusterOperator/redisEnterpriseClusterTemplate.yaml -v service_namespace=service-instances -v instance_name=redis-test | kubectl apply -f-
+```
+
+This will create a cluster and database named _redis-test_.  Once the cluster is ready, its configuration is written to a secret and service resource.
```

Comment:
```suggestion
This will create a cluster and database named `redis-test`.  Once the cluster is ready, its configuration is written to a secret and service resource.
```